crates/sparrow-runtime/src/prepare/slice_preparer.rs

```diff
@@ -222,16 +219,21 @@ mod tests {
         .unwrap();
 
         let sliced_batch = preparer.slice_batch(batch).unwrap();
-        let expected_batch_size = 3;
+        let expected_batch_size = 2;
```

Comment:
Why would the hashing have affected this?

Reply:
See previous comment on slicing.